### ISSUE TITLE
Fatal error: Couldn't find primitive NEW with GCC 14

### ISSUE THREAD

**Issue by diegosardina** [Spivoxity/obc-3#51]
After moving my default compiler in Gentoo from 13 to 14, obc doesn't work anymore (except if compiled with --disable-dynlink).

```
# gcc-config 2
 * Switching native-compiler to x86_64-pc-linux-gnu-14 ...
# ./configure
[...]
# make
[...]
chmod +x oblist
compiler/obc1 -x -g -pl -I lib ./Fac.m >Fac.k
runtime/oblink -g -pl -L lib Fac.k -o Fac.x
for i in obx obxj; do ./$i ./Fac.x 20; done
Fatal error: Couldn't find primitive NEW
Fatal error: Couldn't find primitive NEW
make: *** [Makefile:207: sanity] Error 3
#

```
switching back to gcc 13 makes it working again

**Comment by Spivoxity** [comment]
Please provide complete information when reporting bugs.

This one is fixed on the release-3.3 branch, by updating the config test that determines whether -rdynamic is needed.

**Comment by diegosardina** [comment]
Yes, I was using the master branch, applying [this](https://github.com/Spivoxity/obc-3/commit/acc73561bdee371f5b2409c820cc785db2e7e136) manually solved the problem
